gallery/plot_visualization_utils.py

```diff
@@ -0,0 +1,129 @@
+"""
+=======================
+Visualization utilities
+=======================
+
+This example illustrates some of the utilities that torchvision offers for
+visualizing bounding boxes and segmentation masks.
+"""
+
+
+import torch
+import numpy as np
+import scipy.misc
+import matplotlib.pyplot as plt
+
+import torchvision.transforms as T
+
+
+plt.rcParams["savefig.bbox"] = 'tight'
+
+
+def show(img):
+    plt.imshow(np.transpose(img, (1, 2, 0)))
+    plt.axis('off')
+
+
+####################################
+# Visualizing a grid of images
+# ----------------------------
+# The :func:`~torchvision.utils.make_grid` function can be used to create a
+# tensor that represents multiple images in a grid.  This util requires a single
+# image of dtype ``uint8`` as input.
+
+from torchvision.utils import make_grid
+
+racoon = T.ToTensor()(scipy.misc.face().copy())
+racoon_int = T.ConvertImageDtype(dtype=torch.uint8)(racoon)
+grid = make_grid([racoon_int, racoon_int, racoon_int])
+show(grid)
+
+####################################
+# Visualizing Bounding Boxes
+# --------------------------
+# We can use :func:`~torchvision.utils.draw_bounding_boxes` to draw boxes on an
+# image. We can set the colors, labels, width as well as font and font size !
+# The boxes are in ``(xmin, ymin, xmax, ymax)`` format
+# from torchvision.utils import draw_bounding_boxes
+
+from torchvision.utils import draw_bounding_boxes
+
+
+boxes = torch.tensor([[100, 400, 500, 740], [500, 200, 800, 580]], dtype=torch.float)
+labels = ["grass", "racoon"]
+colors = ["blue", "yellow"]
+result = draw_bounding_boxes(racoon_int, boxes, labels=labels, colors=colors, width=10)
```

Comment:
We also have `fill=True`where boxes are filled. It might be reptitive but good to illustrate?

Also Note to add (if necessary)
flled images are filled by transparency and hence should be saved with `alpha` channel.

Reply:
TBH I think it's best to try to keep the example short. We already illustrate `draw_bounding_boxes` a bunch of times, and showing the effect of every single parameter has a diminished value and can lead to a "too much info" effect. Since there's an obvious link to the docstring of the function, users will be able to discover the rest of the features easily.

Reply:
Sounds fine